sources/packages/forms/src/form-definitions/studentdependantsappealparttime.json

```diff
@@ -0,0 +1,970 @@
+    "name": "studentDependantsAppealPartTime",
+    "path": "studentdependantsappealparttime",
+    "type": "form",
+    "display": "form",
+    "tags": [
+        "common"
+    ],
+    "components": [
+        {
+            "key": "dependantsInformationTitle",
+            "label": "Dependants Information (Part-Time Applications)",
+            "input": false,
+            "tag": "h3",
+            "attrs": [
+                {
+                    "value": "",
+                    "attr": ""
+                }
+            ],
+            "className": "category-header-medium primary-color",
+            "content": "Dependants Information (Part-Time Applications)",
+            "type": "htmlelement",
+            "hideLabel": true,
+            "tags": [],
+            "conditional": {
+                "show": "",
+                "when": null,
+                "eq": ""
+            },
+            "properties": {},
+            "lockKey": true,
+            "isNew": false
+        },
+        {
+            "clearOnHide": false,
+            "key": "dependantsAppealPanel",
+            "input": false,
+            "title": "Dependants Information Part Time Applications Panel",
+            "theme": "default",
+            "tableView": false,
+            "components": [
+                {
+                    "autofocus": false,
+                    "input": true,
+                    "tableView": true,
+                    "inputType": "radio",
+                    "label": "<strong>Do you have any dependants?</strong>",
+                    "key": "hasDependents",
+                    "values": [
+                        {
+                            "value": "yes",
+                            "label": "Yes",
+                            "shortcut": ""
+                        },
+                        {
+                            "value": "no",
+                            "label": "No",
+                            "shortcut": ""
+                        }
+                    ],
+                    "defaultValue": "",
+                    "protected": false,
+                    "fieldSet": false,
+                    "persistent": true,
+                    "hidden": false,
+                    "clearOnHide": true,
+                    "validate": {
+                        "required": true,
+                        "custom": "",
+                        "customPrivate": false
+                    },
+                    "type": "radio",
+                    "labelPosition": "top",
+                    "optionsLabelPosition": "right",
+                    "tags": [],
+                    "conditional": {
+                        "show": "",
+                        "when": null,
+                        "eq": ""
+                    },
+                    "properties": {},
+                    "tooltip": "Dependants represent anyone who you support financially.",
+                    "errorLabel": "Dependent",
```

Comment:
Is the error label as expected?

Reply:
Nice spotting. It is from the original form, but I see we should use dependant instead of dependent. If we use empty string for `errorLabel`, and the result will be the image below.

![image](https://github.com/bcgov/SIMS/assets/148148914/4538f852-1c4d-42ea-a109-be44a0013b79)

If we keep "dependant", then it will say "dependant is required". Please let me know if we should change it.

Reply:
I removed "Dependent" from the "errorLabel", so it will look like the image above now.